README.md

```diff
@@ -1,144 +1,211 @@
-
-- The result of running `terminus cli info`.
-- The output of the issue in debug mode. (Run the command with `--debug` appended to it.)
-- The name and version of the OS you're seeing the issue on.
-
-**If yours is a problem with Terminus itself:**
-
-1. Search [Terminus' issues on GitHub](https://github.com/pantheon-systems/terminus/issues) to see whether another user has reported the same issue you are experiencing.
-2. If the problem you are experiencing is not in the issues, you can open a new issue. Please include the helpful information you have gathered.
-3. If you find your problem in an issue, feel free to add your issue information in the comments and/or subscribe to the issue's notifications.
-
-**If the problem is with the Pantheon platform:**
-
-Head over to [your support tickets in the Pantheon Dashboard](https://dashboard.pantheon.io/users/#support) and submit a new issue ticket. Please include the helpful information you have gathered.
-
-Contributions
--------------
-
-See the [CONTRIBUTING](CONTRIBUTING.md) document.
-
-Credits
--------
-
-Further Reading
----------------
-
-Docs coming soon
-
-If you are looking for the precursor to this project, which is now deprecated, see [https://github.com/pantheon-systems/terminus-deprecated](https://github.com/pantheon-systems/terminus-deprecated)
+If you are planning to run WP-CLI or Drush commands through Terminus, please
+[upload an SSH key](https://pantheon.io/docs/ssh-keys/#add-your-ssh-key-to-pantheon) for the user that will be executing Terminus.
+ 
+## Running
+Commands in Terminus follow a predictable pattern:
+```bash
+terminus command:subcommand:subcommand param param --option=value --option
+```
+- `terminus` is the name of the application.
+- `command[:subcommand[:subcommand]]` is the name of the command to run. Terminus commands may consist of only a `command`.
+- `param` are parameters, and are almost always required for operating the command.
+- `option` are options, and are never required to run a command. They may or may not require a value.
+
+## Runtime Configuration
+### Setting default user, site, environment, etc. and Dotenv
+Terminus can use certain environment variables to set certain default values when invoking commands. Check the 
+`config/constants.yml` file for the names of variables which can be set. Do not alter this file - add global settings to
+your `~/.terminus/config.yml` file or export them in your terminal client.
+
+Terminus also has built-in support for [PHP Dotenv](https://github.com/vlucas/phpdotenv), which provides an alternative
+to manually exporting environment variables. This involves defining environment variables within a `.env` file that
+Terminus will automatically use when invoked within that working directory.
+
+## Known Issues/Limitations
+- Terminus will not offer you options for selection when parameters are not provided. This will be added in the future.
+ 
+## Index
```

Comment:
I think this section is pretty optional overall for terminus. It's origin was to help people navigate the monorepos.

This is mostly (only?) useful for contributors so if we keep it, I think it makes sense under the Developing & Contributing section with the heading: Repository Index to disambiguate it from the installed filesystem layout?